tensorboard/webapp/persistent_settings/_data_source/types.ts

```diff
@@ -50,7 +64,7 @@ export declare interface BackendSettings {
  */
 export interface PersistableSettings {
   scalarSmoothing?: number;
-  tooltipSortString?: string;
+  tooltipSortString?: TooltipSort;
```

Comment:
tooltipSortString is no longer a string, so perhaps a name change is appropriate.

Reply:
That's an excellent point! I'll change this to just `tooltipSort`